singolo/sass/header/_burger-menu.scss

```diff
@@ -0,0 +1,43 @@
+.burger-menu{
+        display: none;
+        @media (max-width: 767px) {
+    display: flex;
+    width: 25px;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 20px;
+    margin-left: 20px;
+    margin-right: calc(50vw - 46px - 45px);
+    
+    }
+        }
+.open{
+        transition: all 1s ease-in-out;
+        justify-content: flex-start;
+       flex-direction: column;
+        height: 100vh;
+        width: 75vw;
+        box-shadow: 50vw 0px 0px rgba(0, 0, 0, 0.1);
+        background: #2D303A;
+      position: absolute;
+        z-index: 100;
+        overflow: hidden;
+       
+}
+
+.burger-menu{
+        transition: all 1s ease-in-out;
+        transform: rotate(0deg); 
+}
+.click {
```

Comment:
Is it an active state for the burger menu?  Mabe, it can be as a modifier?

Reply:
fixed